rita/payment_controller/src/lib.rs

```diff
@@ -299,7 +322,7 @@ mod tests {
     #[test]
     fn test_multi_make_payments() {
         // mock neighbour
```

Comment:
It's `neighbor` or `neighbour`, let's use only one and be consistent or stupid bugs will befall us. Personally I'd go with `neighbor`

Reply:
Changed all usages to neighbor, although babel uses neighbour